pwem/protocols/protocol_particles.py

```diff
@@ -313,7 +313,16 @@ def _useCTF(self):
 
     # ------ Methods for Streaming extraction --------------
     def _isStreamClosed(self):
-        return self.micsClosed and self.ctfsClosed and self.coordsClosed
+        return self.coordsClosed
+
+    def _isAllMicsProcessed(self):
```

Comment:
_areAllMicsProcessed ?

Reply:
True it was a misspelling, I already change it.